src/Hamiltonians/TwoBodyCorrelation.jl

```diff
@@ -1,18 +1,32 @@
 # Methods that need to be implemented:
 #   •  starting_address(::AbstractHamiltonian) - not needed
 """
-    G2Correlator(d::Int) <: AbstractHamiltonian{ComplexF64}
+    G2Correlator(d::Int,c=:cross) <: AbstractHamiltonian{ComplexF64}
 
-Two-body correlation operator representing the inter-component density-
-density correlation at distance `d` of a two component system
+Two-body correlation operator representing the density-density
+correlation at distance `d` of a two component system
 in a momentum-space Fock-state basis.
 It returns a `Complex` value.
-It currently only works on [`BoseFS2C`](@ref).
-
 
+Correlation across two-component:
+```math
+\\hat{G}^{(2)}(d) = \\frac{1}{M}\\sum_{spqr=1}^M e^{-id(p-q)2π/M} a^†_{s} b^†_{p}  b_q a_r δ_{s+p,q+r}
+```
+Correlation within a single component:
 ```math
-\\hat{G}_2(d) = \\frac{1}{M}\\sum_{spqr=1}^M e^{-id(p-q)2π/M} a^†_{s} b^†_{p}  b_q a_r δ_{s+p,q+r}
+\\hat{G}^{(2)}(d) = \\frac{1}{M}\\sum_{spqr=1}^M e^{-id(p-q)2π/M} a^†_{s} a^†_{p}  a_q a_r δ_{s+p,q+r}
 ```
+# Arguments
+- `d::Integer`: the distance between two particles.
+- `c`: possible instructions: `:cross`: default instruction, computing correlation between particles across two components;
```

Comment:
There is probably a better word for this than instruction, but I don't know what it is.